docker-compose.yml

```diff
@@ -72,6 +72,21 @@ services:
       - '1977:1977'
     networks:
       - default
+  pattern-atlas-ui:
+    image: patternpedia/patternrepo-ui:latest
```

Comment:
change from latest to the last version, I guess v1.2.0

Reply:
Done the last 2 changes